src/config/serverConfigDefaults.ts

```diff
@@ -52,6 +52,18 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     skin_show_web_api_tab:true,
     skin_show_tweet_button:false,
     skin_show_tissue_image_tab:true,
+    quick_search_show: false,
+    quick_search_gene_query_cancer_study_list:
+        "laml_tcga_pan_can_atlas_2018,acc_tcga_pan_can_atlas_2018,blca_tcga_pan_can_atlas_2018," +
```

Comment:
fyi you can now use `  ` template string do multiline strings in javascript instead of annoying concats